src/css/intlTelInput.scss

```diff
@@ -229,6 +229,12 @@ $flagsImageExtension: "png" !default;
     height: 100%;
     opacity: 0;
   }
+
+  // if container option is set, increase z-index to prevent display issues
+  &.iti-container {
+    position: absolute;
+    z-index: 1060;
```

Comment:
Ok I'm down with copying Bootstrap's z-index value for now (for lack of any other ideas), but let's at least add a comment explaining why we went with this particular value for future reference.